e2e/cypress/tests/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -136,7 +136,7 @@ context('ldap', () => {
             cy.get('#addGroupsToTeamToggle').scrollIntoView().click();
             cy.get('#multiSelectList').should('be.visible');
             cy.get('#multiSelectList>div').children().eq(0).click();
-            cy.uiGetButton('Add').click();
+            cy.uiGetButton('Add').click().wait(TIMEOUTS.ONE_SEC);
```

Comment:
can we wait on Network call?
or
after click, 
`cy.get(locator).should('be.visible');`